lib/govuk_app_config/govuk_content_security_policy.rb

```diff
@@ -124,15 +125,19 @@ def self.build
     policies.map { |str| str.join(" ") }.join("; ") + ";"
   end
 
+  def self.govuk_domains
+    Rails.env.production? ? PROD_GOVUK_DOMAINS : DEV_GOVUK_DOMAINS
+  end
```

Comment:
I don't think we should use the env as an indicator of what domains we'd expect. Rails production mode shouldn't be exclusively for running on production servers and it is used as the environment for running end-to-end tests (which is a reason why we should be doing https://github.com/alphagov/govuk_app_config/blob/243eaca4a23a1e3c4fb835d81f668e3c533b49e3/lib/govuk_app_config/govuk_content_security_policy.rb#L132 via an env var rather than hardcoding).

I also think we'll need `*.publishing.service.gov.uk` in all environments (e.g accessing assets or using [live static](https://github.com/alphagov/government-frontend/blob/83ab8d4fe9df05753b6821d16dde85ce3c447ae5/startup.sh#L15)) 

How much of a concern do you think it is to just allow `*.dev.gov.uk` globally? Some of our options around this are using Plek or the GOVUK_APP_DOMAIN env vars. Hopefully there aren't real needs for localhost in either dev or prod.